experimentation.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Experimentation Support'
+description: 'This tutorial explains how to do experiments in your Wyvern Pipelines.'
+icon: 'flask'
+---
+
+Wyvern experimentation framework is designed to be agnostic to your preferred experimentation provider. It is designed
+to be flexible and allow you to use any experimentation provider you want. In this tutorial, we will show you how to
+use Wyvern with [Eppo](https://www.geteppo.com/) as your experimentation provider. Currently, Wyvern only supports Eppo
+as an experimentation provider, but we are working on adding support for other providers.
+
+## Prerequisites
+- Install `wyvern-ai >= 0.0.6` or make sure you have it installed.
```

Comment:
can just point to https://docs.wyvern.ai/quickstart#install-wyvern

Reply:
![image](https://github.com/Wyvern-AI/docs/assets/26116031/82e4fee7-26e8-4550-82ae-9a10bac1bf6b)